images/virtualization-artifact/pkg/controller/vm/vm_webhook.go

```diff
@@ -162,48 +162,48 @@ func newIPAMVMValidator(ipam internal.IPAM, client client.Client) *ipamVMValidat
 }
 
 func (v *ipamVMValidator) validateCreate(ctx context.Context, vm *v1alpha2.VirtualMachine) (admission.Warnings, error) {
-	claimName := vm.Spec.VirtualMachineIPAddressClaim
-	if claimName == "" {
-		claimName = vm.Name
+	vmipName := vm.Spec.VirtualMachineIPAddress
+	if vmipName == "" {
+		vmipName = vm.Name
 	}
 
-	claimKey := types.NamespacedName{Name: claimName, Namespace: vm.Namespace}
-	claim, err := helper.FetchObject(ctx, claimKey, v.client, &v1alpha2.VirtualMachineIPAddressClaim{})
+	vmipKey := types.NamespacedName{Name: vmipName, Namespace: vm.Namespace}
+	vmip, err := helper.FetchObject(ctx, vmipKey, v.client, &v1alpha2.VirtualMachineIPAddress{})
 	if err != nil {
-		return nil, fmt.Errorf("unable to get Claim %s: %w", claimKey, err)
+		return nil, fmt.Errorf("unable to get vmip %v: %w", vmip, err)
 	}
 
-	if claim == nil {
+	if vmip == nil {
 		return nil, nil
 	}
 
-	if vm.Spec.VirtualMachineIPAddressClaim == "" {
-		return nil, fmt.Errorf("VirtualMachineIPAddressClaim with the name of the virtual machine"+
-			" already exists: explicitly specify the name of the VirtualMachineIPAddressClaim (%s)"+
-			" in spec.virtualMachineIPAddressClaim of virtual machine", claim.Name)
+	if vm.Spec.VirtualMachineIPAddress == "" {
+		return nil, fmt.Errorf("VirtualMachineIPAddress with the name of the virtual machine"+
+			" already exists: explicitly specify the name of the VirtualMachineIPAddress (%s)"+
+			" in spec.virtualMachineIPAddress of virtual machine", vmip.Name)
```

Comment:
Just add more info about "what you get if follow my advice".
```suggestion
	// VM is created without ip address, but ip address resource is already exists.
	if vm.Spec.VirtualMachineIPAddress == "" {
		return nil, fmt.Errorf("VirtualMachineIPAddress with the name of the virtual machine"+
			" already exists: set spec.virtualMachineIPAddress field to %s to use IP %s", vmip.Name, vmip.Status.Address)
```